### PR TITLE
Warning cleaning: unused imports and definitions.

### PR DESCRIPTION
**Pull request by quasicomputational**

### CODE REVIEW

**Comment by ocharles** [comment]
Thanks, I'd been meaning to do this.

File: lib/DhallToCabal.hs [hunk 1]
```diff
@@ -46,13 +46,9 @@ import Data.Maybe ( fromMaybe )
 import Data.Monoid ( (<>) )
 import Formatting.Buildable ( Buildable(..) )
 
-import qualified Data.ByteString.Lazy as LazyByteString
```
Comment (ocharles) [inline lib/DhallToCabal.hs]: Can we drop a whole `bytestring` dependency?

Comment (quasicomputational) [reply lib/DhallToCabal.hs]: Sadly no. Cabal 2.2's parser consumes bytestrings.

Comment (ocharles) [reply lib/DhallToCabal.hs]: Not the end of the world, it's a base package after all.

**Comment by quasicomputational** [comment]
Would it be OK if I turned the warning level up? I've found that `-Weverything` only has a few that need turning off, and the rest are at worst harmless and at best seriously useful.

**Comment by ocharles** [comment]
Always in favour of more warnings!

**Comment by quasicomputational** [comment]
OK! I'll go ahead with that on this branch.

**Comment by ocharles** [comment]
Perfect, thanks!

**Pull request merged**